Siminar7/logger.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime as dt
+from time import time
+
+def id_logger(ide):
+    time = dt.now().strftime('%d/%m/%y %H:%M')
+    with open('log.csv', 'a') as file:
+        file.write('{};ide;{}\n'
+                    .format(time, ide))
+
+
+def name_logger(name):
+    time = dt.now().strftime('%d/%m/%y %H:%M')
+    with open('log.csv', 'a') as file:
+        file.write('{};name;{}\n'
+                    .format(time, name))
+
+
+def birthday_logger(birthday):
+    time = dt.now().strftime('%d/%m/%y %H:%M')
+    with open('log.csv', 'a') as file:
+        file.write('{};birthday;{}\n'
+                    .format(time, birthday))  
+
+
+def work_logger(work):
+    time = dt.now().strftime('%d/%m/%y %H:%M')
+    with open('log.csv', 'a') as file:
+        file.write('{};work;{}\n'
+                    .format(time, work))
+
+
+
+def telephone_logger(telephone):
+    time = dt.now().strftime('%d/%m/%y %H:%M')
+    with open('log.csv', 'a') as file:
+        file.write('{};telephone;{}\n'
+                    .format(time, telephone))
+
+
+def log_collection(ide, name, birthday, work, telephone):
+    return(id_logger(ide), name_logger(name), birthday_logger(birthday), work_logger(work), telephone_logger(telephone))
+
+
+def save(ide, name, birthday, work, telephone):
```

Comment:
```suggestion
def save(*args):
```
И ниже можно использовать цикл для записи, чуть проще получится.